src/NuGet.Services.BasicSearch/Web.config

```diff
@@ -14,10 +14,10 @@
     <sessionState mode="Off" />
   </system.web>
   <appSettings>
-    <!--
-    <add key="Local.Lucene.Directory" value="C:\nuget\SearchData\Lucene-v2v3" />
+
+    <add key="Local.Lucene.Directory" value="C:\nuget\newLucene" /><!--"C:\nuget\SearchData\Lucene-v2v3" /-->
```

Comment:
Reverted.

---
In reply to: [75552446](https://github.com/NuGet/NuGet.Services.Metadata/pull/87#discussion_r75552446) [](ancestors = 75552446)